src/page-header/index.md

```diff
@@ -0,0 +1,19 @@
+# What is a page header?
+
+Headers are the big, banner-like images that adorn the tops of game pages. They are optional but highly, highly recommended. Headers make for a splendid looking page.
```

Comment:
Mention they are rectangles "Headers are the big, banner-like rectangle images that adorn the tops of game pages."